kernels/volk/volk_16i_branch_4_state_8.h

```diff
@@ -98,73 +94,71 @@ volk_16i_branch_4_state_8_a_ssse3(short* target, short* src0, char** permuters,
   xmm8 = _mm_load_si128((__m128i*)permuters[2]);
   xmm10 = _mm_load_si128((__m128i*)permuters[3]);
 
-  for(; i < bound; ++i) {
-    xmm5 = _mm_load_si128(p_src0);
-    xmm0 = _mm_shuffle_epi8(xmm5, xmm0);
-    xmm6 = _mm_shuffle_epi8(xmm5, xmm6);
-    xmm8 = _mm_shuffle_epi8(xmm5, xmm8);
-    xmm10 = _mm_shuffle_epi8(xmm5, xmm10);
+  xmm5 = _mm_load_si128(p_src0);
+  xmm0 = _mm_shuffle_epi8(xmm5, xmm0);
+  xmm6 = _mm_shuffle_epi8(xmm5, xmm6);
+  xmm8 = _mm_shuffle_epi8(xmm5, xmm8);
+  xmm10 = _mm_shuffle_epi8(xmm5, xmm10);
 
-    p_src0 += 4;
+  p_src0 += 4;
```

Comment:
this line can be removed too

Reply:
Yeah, it is just unused (!) I will also delete those then: 
`p_target = (__m128i*)target; 
  p_src0 = (__m128i*)src0;`